src/slices/StakeThunk.js

```diff
@@ -3,17 +3,14 @@ import { addresses, Actions } from "../constants";
 import { abi as ierc20Abi } from "../abi/IERC20.json";
 import { abi as OlympusStaking } from "../abi/OlympusStakingv2.json";
 import { abi as StakingHelper } from "../abi/StakingHelper.json";
-import { clearPendingTxn, fetchPendingTxns, getStakingTypeText } from "./PendingTxns.actions";
-import { getBalances } from "./App.actions";
+import { clearPendingTxn, fetchPendingTxns, getStakingTypeText } from "./PendingTxnsSlice";
```

Comment:
I'm a big fan of these being explicit actions defined on the slice, so we're more tightly coupled with the the sideeffects of calling an action.